src/app/components/ContentMessage/index.tsx

```diff
@@ -1,15 +1,92 @@
+import { t } from "i18next";
+
+/* eslint-disable  @typescript-eslint/no-explicit-any */
 type Props = {
   heading: string;
-  content: string;
+  content:
```

Comment:
I believe ContentMessage should be a generic component and the content type should not be modified here